Pipfile

```diff
@@ -17,13 +17,13 @@ djangorestframework-csv = "==2.1.0"
 pytz = "==2019.2"
 django-prometheus = "==1.0.15"
 prometheus-client = "==0.7.1"
-urllib3 = "==1.25.9"
+urllib3 = "==1.26.6"
 Jinja2 = "==2.11.3"
-watchtower = "==0.6.0"
-boto3 = "==1.9.216"
+watchtower = "==1.0.6"
```

Comment:
Looks like `watchtower` is a major release bump. Do you know what breaking changes may have been introduced with v1?

Reply:
@coderbydesign , judging by the [change log](https://github.com/kislyuk/watchtower/blob/5002639093154d29fdeb1bbeb8a2fb30c3b7f48e/Changes.rst) for `watchtower`, the updates consisted of a bug fixes, a couple of new functions/features, and the drop of Python 2 support. I don't believe there any breaking changes.